docs/react-integration.md

```diff
@@ -295,16 +295,16 @@ Since `onRender` is part of the rendering cycle of `GridRow`, rather than `TodoV
 Or, we can create an in-line anonymous observer using [`<Observer />`](https://github.com/mobxjs/mobx-react#observer):
 
 ```javascript
-const TodoView = observer(({ todo }: { todo: Todo }) =>
+const TodoView = observer(({ todo }: { todo: Todo }) => {
```

Comment:
Well, the `observer` shouldn't be here at all if understand correctly what this example is trying to demonstrate.

Reply:
Imo it tries to show that even though `todo.title` appears in `observer`, it's not actually evaluated during it's execution and therefore either `GridRow` must be observer or if thats not possible, the callback must be wrapped in `<Observed>`.

Reply:
Ah right, I missed the whole point with callback here :) Cool.